src/zcl_abapgit_objects.clas.abap

```diff
@@ -388,6 +397,11 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
       EXPORTING
         iv_total = lines( lt_tadir ).
 
+    lt_items = map_tadir_to_items( lt_tadir ).
+
+    check_objects_locked( iv_language = zif_abapgit_definitions=>gc_english
```

Comment:
english? this looks suspicious

Reply:
well, it doesn't matter and it's the same as e.g. here
https://github.com/larshp/abapGit/blob/145a82e9eff61961a00f836c7ccd06b0e55dd981/src/zcl_abapgit_objects.clas.abap#L614-L615